app/uk/gov/hmrc/cataloguefrontend/platforminitiatives/PlatformInitiativesListPage.scala.html

```diff
@@ -29,7 +29,7 @@
 @standard_layout(s"Platform Initiatives", "platforminitiatives") {
 <header>
     <h1>Platform Initiatives</h1>
-    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript" src="@controllers.routes.Assets.versioned("loader.js")"></script>
```

Comment:
can we rename the file to include `charts` or put in a `charts` folder.
Also include the version in the file-name so we know what it is.